.changeset/moody-crabs-march.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-vault': minor
```

Comment:
Think both of these changes should be `patch` instead as the `vault` plugin is <1.x